header.php

```diff
@@ -23,14 +23,14 @@ function mp_ssv_get_main_nav_bar()
 {
     ob_start();
     if (!is_home() && !is_front_page()) {
-        ?><a class="brand-logo" href="<?php echo esc_url(home_url('/')); ?>" rel="home">
+        ?><a class="nav-logo" href="<?php echo esc_url(home_url('/')); ?>" rel="home">
         <?php
         $logoPath = get_theme_mod('navbar_logo', false);
         if (!$logoPath) {
             $logoPath = get_template_directory_uri() . '/images/logo.svg';
         }
         ?>
-        <img src="<?= $logoPath ?>" alt="<?php bloginfo('name'); ?>" style="height:60px"/></a>
+        <img src="<?= $logoPath ?>" alt="<?php bloginfo('name'); ?>" style="height:55px"/></a>
```

Comment:
Can this be handled with a CSS class (that way we can use the  $navbar-height and $navbar-height-mobile parameters).

Reply:
Perhaps if I stop being lazy